x/epoching/keeper/keeper.go

```diff
@@ -0,0 +1,193 @@
+package keeper
+
+import (
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	db "github.com/tendermint/tm-db"
+)
+
+const (
+	DefaultEpochActionID = 1
+	DefaultEpochNumber   = 0
+)
+
+var (
+	NextEpochActionID      = []byte{0x49}
```

Comment:
Any reason for starting at 0x49?

Reply:
no reason, switched to 10 to start. do you have a preference?